frontend/src/model/Settings.ts

```diff
@@ -0,0 +1,13 @@
+class Settings {
+  turnLength: number;
+  tileAmount: number;
+  boardSize: number;
+
+  constructor(turnLength: number, tileAmount: number, boardSize: number) {
+    this.turnLength = turnLength;
+    this.tileAmount = tileAmount;
+    this.boardSize = boardSize;
```

Comment:
It is a good idea to prevent situations with non-positive numbers (or even with positive floats)